enterprise/admin/__init__.py

```diff
@@ -90,6 +90,7 @@ class EnterpriseCustomerIdentityProviderInline(admin.StackedInline):
 
     model = EnterpriseCustomerIdentityProvider
     form = EnterpriseCustomerIdentityProviderAdminForm
+    extra = 0
```

Comment:
Not really sure what's the job of this flag, can you explain it in the PR and if possible, add a comment in the code too?

Reply:
This would allow nested inlines. So we can link multiple IDPs to enterprise_customer easily from Django-admin.